library/payment/credit_card.py

```diff
@@ -25,3 +25,13 @@ def check_validity(self) -> bool:
             and self._valid_date > datetime.now()
             and self._cvv != "000"
         )
+
+    def pay(self, fee: float) -> bool:
```

Comment:
We like this very much, but shouldn't that be a method for both credit card and paypal?

Reply:
It might make sense to have a common abstract superclass `PaymentMethod` that implements pay methods individually. This could then be structured into one file.